doc/doxygen/src/Glossary.dox

```diff
@@ -332,10 +332,11 @@
  * It is common for am @ref asset_management_system to allow an @ref entity
  * version managed. Some systems also allow the concept of what we call a
  * 'meta-version'. For example "latest" or "latestApprovedForLighting".  These
- * are version strings that need to be further resolved using @ref
- * openassetio.hostApi.Manager.Manager.finalizedEntityVersion
- * "finalizedEntityVersion()" before they can be assumed to be
- * pointing to a specific 'concrete' version of an entity.
+ * are version strings that can't be assumed to be stable. Traits may
+ * exists within domain specific uses of OpenAssetIO (such as in
+ * OpenAssetIO-MediaCreation) that allow the
```

Comment:
[Consider] Should this link to the mediacreation repo?

Reply:
Added link [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/982/commits/99e7cbf5a694157fe5b33fec5775ebc3907b3c0a).